### DISCUSSION TITLE
Accept from different release groups, even with same quality

### DISCUSSION THREAD

**Discussion by Koyan99** [Flexget/Flexget#3797] [Q&A]
Hi!

I'm a new user, no experience with python, and I'm trying to do something that seems very simple to me, but I can't figure out how to do it. I've searched all through the documentation and all over the forum here, but can't find the solution. Can you please help me out?

I just want very simply to download all episodes of a show, filtered only on "quality: 1080p webdl h264". I don't care if I've downloaded that episode before in the same quality, I want to download them all. But no matter what I do Flexget keeps rejecting "by series plugin because quality already downloaded"

I've tried adding "disable: seen" and "tracking: no", I've experimented with upgrade, qualities, propers, but the later matches keep getting rejected.

How can I disable this filtering?

**Comment by lamski** [comment]
Maybe you can experiment with the `regexp` plugin?
